app/commands/top.py

```diff
@@ -0,0 +1,60 @@
+from app.common.database.repositories import users, scores
+from app.common.constants import Mods
+from app.objects import Context
+from discord import Embed
+from discord import Color
+import config
+import app
+
+
+@app.session.commands.register(["top"])
+async def top(context: Context):
+    if not (user := users.fetch_by_discord_id(context.message.author.id)):
+        await context.message.channel.send("You don't have an account linked!")
+        return
+    mode = user.preferred_mode
+
+    modes = {"std": 0, "taiko": 1, "ctb": 2, "mania": 3}
+    if len(context.args):
+        if context.args[0] in modes:
+            mode = modes[context.args[0]]
+        else:
+            await context.message.reply(
+                f"Wrong mode! Available modes: {', '.join(modes.keys())}"
+            )
+            return
+
+    user_scores = scores.fetch_top_scores(user_id=user.id, mode=mode)[:10]
```

Comment:
I would actually add a `limit` parameter here. Let me implement that really quick...

Reply:
@kanaarima can you sync the fork and add `limit=10`?